tardis/apps/publication_workflow/components/PublicationCard.jsx

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import styled from '@emotion/styled';
+import { Button } from 'react-bootstrap';
+import moment from 'moment';
+
+const Hover = styled.card({
+  ':hover': {
+    transform: 'scale(1.01)',
+    boxShadow: '0 10px 20px rgba(0,0,0,.12), 0 4px 8px rgba(0,0,0,.06)',
+  },
+});
+
+const PublicationCard = ({
+  publicationType, data, handleDelete, handleRetract, handleResume, colNum,
+}) => (
+  <Fragment>
+    <div className={`col-md-${colNum} col-xs-12 pb-3`}>
+      <Hover className="card h-100">
+        <div className="card-body">
+          {/* eslint-disable-next-line consistent-return */}
+          {(() => {
+            switch (publicationType) {
+              case 'draft': return <span className="badge badge-secondary">Draft Publication</span>;
+              default: return <span className="badge badge-success">Publication Released</span>;
+              case 'scheduled': return <span className="badge badge-info">Publication Scheduled</span>;
+              case 'retracted': return <span className="badge badge-danger">Publication Retracted</span>;
+            }
+          })()}
+
+          <h5 className="card-title">{data.title}</h5>
+          <p className="card-text text-muted">
+            {data.description}
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 32 spaces but found 16.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                                {data.description}
```